src/main/kotlin/probation/model.kt

```diff
@@ -1,12 +1,16 @@
 package uk.gov.justice.hmpps.architecture.probation
 
+import com.structurizr.model.CreateImpliedRelationshipsUnlessAnyRelationshipExistsStrategy
 import com.structurizr.model.Model
 
 import uk.gov.justice.hmpps.architecture.prison.NOMIS
-import uk.gov.justice.hmpps.architecture.prison.OffenderAllocationManager
+import uk.gov.justice.hmpps.architecture.prison.OffenderManagementInCustody
 import uk.gov.justice.hmpps.architecture.shared.Tags
 
 fun probationModel(model: Model) {
+  model.setImpliedRelationshipsStrategy(
+      CreateImpliedRelationshipsUnlessAnyRelationshipExistsStrategy())
```

Comment:
I'm curious, what does this do?

Reply:
It's a way to automatically add relationships to a system if its container is used.

This line adds a relationship between a system and a container:
```kotlin
ndmis.uses(omic.allocationManager, "sends extracts containing service user allocation to", "email")
```

The strategy above will automatically add a relationship between `ndmis` and `omic.system`.
The goal is to have the connection there for the higher level diagrams.

https://github.com/structurizr/java/blob/v1.4.7/structurizr-core/src/com/structurizr/model/CreateImpliedRelationshipsUnlessAnyRelationshipExistsStrategy.java